src/ch.c

```diff
@@ -273,25 +330,24 @@ int main(int argc, char *argv[])
 
         /* --------------------------------------------------------*\
    	                    Clipping  mag of complex signal
-   	    \*---------------------------------------------------------*/
+        \*---------------------------------------------------------*/
 
         for(i=0; i<BUF_N; i++) {
             float mag = sqrt(ch_in[i].real*ch_in[i].real + ch_in[i].imag*ch_in[i].imag);
-            //fprintf(stdout, "%f\n",mag);
             float angle = atan2(ch_in[i].imag, ch_in[i].real);
             if (mag > clip) {
               mag = clip;
               nclipped++;
             }
             tx_pwr += mag*mag;
-            if (mag > peak) peak = mag;
+            if (mag > peak) { peak = mag; /*fprintf(stderr, "%f\n",mag);*/ }
```

Comment:
Does the commented code here need to be removed?

Reply:
I've actually been using it to debug just today over in https://github.com/drowe67/codec2/pull/274 !  We can remove it a little bit down the track.  I ended up putting a Tx BPF into 2020 to prevent some weirdness in this magnitude :thinking: